airflow/providers/postgres/hooks/postgres.py

```diff
@@ -320,6 +320,32 @@ def _generate_insert_sql(
 
         return sql
 
+    def ingest_embedding(self, table: str, input_data: Iterable[tuple[str, list[float]]], vector_size: int) -> None:
+        """
+        Store embedding vector in Postgres table.
+
+        :param table: The Name of the table
+        :param input_data: Iterable containing tuples of input data and corresponding embedding vectors.
+        :param vector_size: The size of vector. The maximum dimensions can be 2,000
+        """
+        from pgvector.psycopg import register_vector
+        from psycopg2 import sql
+
+        self.conn.execute("CREATE EXTENSION IF NOT EXISTS vector")
```

Comment:
```suggestion
```

AFAIK, before Postgres 13 CREATE EXTENSION required superuser permissions, in PG 13+ trusted extension could be installed by someone with appropriate CREATE, however it is not a case of [pgvector](https://github.com/pgvector/pgvector/issues/288)

So let keep create extensions for DBAs or for some one with appropriate permissions. We should not force users to use SU in their databases.